docs/dashboard/access-roles.md

```diff
@@ -0,0 +1,53 @@
+<!--
+title: Serverless Dashboard - Access Roles
+menuText: Access Roles
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Access Roles
+
+You can use the Serverless Framework Dashboard to set up an AWS Access Role to help you secure your service deployments on AWS by enabling the Serverless Framework to issue temporary AWS Access Keys to deploy your services to AWS.
+
+If an Access Role is not configured, the Serverless Framework service will use AWS Access Keys stored in [environment variables](https://serverless.com/framework/docs/providers/aws/guide/credentials/) or [AWS Profiles](https://serverless.com/framework/docs/providers/aws/guide/credentials/) to deploy your service. 
+
+With AWS Access Roles the AWS Access Keys are generated by Serverless Framework on every command and the credentials expire after one hour.  The Serverless Framework leverages AWS Security Token Service and the AssumeRole API to automate creating and usage of temporary credentials, so your developers can stay productive and work securely without doing this manually.
+
+If you do not use the Serverless Framework Dashboard to set up an AWS Access Role, then you will need to configure your Serverless Framework open source CLI to use the AWS Access Keys stored in [environment variables](https://serverless.com/framework/docs/providers/aws/guide/credentials/) or [AWS Profiles](https://serverless.com/framework/docs/providers/aws/guide/credentials/). 
+
+## Link your AWS Account
+
+1. Open https://dashboard.serverless.com/
```

Comment:
We should make this link clickable:

```
1. Open [the Dashboard](https://dashboard.serverless.com/)
```

Reply:
Isn't there a direct link? This way we wouldn't need to describe the steps and therefore don't run into future problems when the steps change.

Reply:
Added link. Not sure if we can do a direct link. @skierkowski ?

Reply:
:+1: Thanks